picard/ui/options/tags.py

```diff
@@ -151,8 +152,9 @@ def completer_activated(self, text):
         current = input_field.text()
         i = input_field.cursorPosition()
         p = len(self.completer.completionPrefix())
-        input_field.setText("%s%s %s" % (current[:i - p], text, current[i:]))
-        input_field.setCursorPosition(i - p + len(text) + 1)
+        start_text = current[:i - p].rstrip()
+        input_field.setText("%s %s, %s" % (start_text, text, current[i:].lstrip()))
+        input_field.setCursorPosition(len(start_text) + len(text) + 3)
```

Comment:
Why +3? Maybe add a comment explaining this so that this doesn't go out of date in the future.

Reply:
The 3 are the extra characters added: It's the length of the existing text before autocomplete + length of the autocompleted text + the three characters (space before the autocomplete, comma and space after).

I tried to make this whole code more readable in https://github.com/metabrainz/picard/pull/1350/commits/feca8d80b38c2ac9cc7173bfc7f54ea36d9a39c6 . This also gets rid of the magic constant.